### ISSUE TITLE
Request: adding version number to readme

### ISSUE THREAD

**Issue by jedai47** [hink/ip2asn#1]
Hello:
As i am using the release the version is 1.0. Is it the same from the current one master ? Thanks a lot

**Comment by hink** [comment]
Yes, This project has been somewhat dead as it was done as a learning excersise for Go.

**Comment by hink** [comment]
@jedai47 Better late than never. I totally refactored the tool. It now uses regex and supports IPv4 and IPv6. You can feed it any text input now and it will extract IP addresses.

I have added the version number in the Readme and will be using tags if i have any future iterations.
